app/build.gradle

```diff
@@ -84,7 +84,7 @@ android {
 
     splits {
         abi {
-            enable true
+            enable !project.hasProperty("skipSplitApk")
```

Comment:
@pocmo I had to disable APK splitting based on whether we're doing unit tests.
I'm not sure what the IDE experience for this is (I ran the tests in `androidTest`, but I think that those are "connectedAndroidTests", not "unitTests").
How do you run the unit tests locally in-IDE? I'd like to test it before I land this :)

Reply:
Oh weird, that wasn't mentioned in the apk split docs. Did you find others do that?

For running unit tests I usually right click on the `test` package and select "run tests". This also works for sub packages and single test classes or methods.

Reply:
Is there a `test` package in `reference-browser`? I can't find it :thinking: 
FWIW, I made a dummy android package and set up APK splitting, and testing worked (even with split apks set up).
However, it might be worth removing this wrinkle from our gradle config in an `gradle`-specific follow-up ticket, since I'm guessing it'll take a while to figure out

Reply:
Um, heh, yeah there's none yet actually. We never added unit tests. :D